cmd/kwild/config/test_data/config.toml

```diff
@@ -96,14 +96,15 @@ hostname = ""
 ###                Snapshot store Config Options                    ###
 #######################################################################
 [app.snapshots]
+
 # Toggle to enable snapshot store
-# This would snapshot the application state at every snapshot_heights blocks
+# This would snapshot the application state at every recurring_height blocks
 # and keep max_snapshots number of snapshots in the snapshot_dir
 # Application state includes the databases deployed, accounts and the Validators db
 enabled = false
 
 # The height at which the snapshot is taken
-snapshot_heights = 100000
+recurring_height = 100000
```

Comment:
Do we need to change the CLI flags too?

Also, could either you or @charithabandi add a pr for docs?

Reply:
It was already changed everywhere else in this repo.  Not sure about docs though

Reply:
Just checked and docs repo has correct setting.  These two toml files where the only place `snapshot_heights` was left.